### ISSUE TITLE
jetson-nano-emmc does not boot.

### ISSUE THREAD

**Issue by pgils** [balena-os/balena-jetson#129]
balena-jetson version: ` v2.67.2+rev1`
machine: `jetson-nano-emmc`

Booting a `jetson-nano-emmc` device results in this error:
```
ERROR: Could not find a resin image of any sort.
Loading resinOS_uEnv.txt from device partition 0xC
Loading extra_uEnv.txt from device partition 0xC
Loading bootcount.env from device partition 0xC
No bootcount.env file. Setting bootcount=0 in environment
Unknown command 'fsuuid' - try 'help'
```
And the `root` kernel parameter is unset:
```
append: ... root=UUID= ro
```

I flashed the devices with balena-os/jetson-flash@c71c025 which uses the r32.4.3 BSP.

**Comment by pgils** [comment]
@acostach If you can confirm this, i've proposed a fix in #130 that works for me.

**Comment by pgils** [comment]
This is now fixed by https://github.com/balena-os/balena-jetson/commit/8c700eca98c48177573b916c3b13750bd9a089df
